datasets/inspecties/dataset.json

```diff
@@ -0,0 +1,281 @@
+{
+  "id": "inspecties",
+  "type": "dataset",
+  "auth": "FP/MDW",
+  "authorizationGrantor": "Directie Stadsbeheer",
+  "theme": [
+    "duurzaamheid en milieu",
+    "Ruimte en Topografie"
+  ],
+  "homepage": "https://data.amsterdam.nl",
+  "owner": "Gemeente Amsterdam, Stadsbeheer",
+  "dateModified": "2021-04-27",
+  "spatialDescription": "Gemeente Amsterdam",
+  "version": "1.0.0",
+  "title": "Inspecties",
+  "language": "nl",
+  "dateCreated": "",
+  "license": "Anders, Niet Open",
+  "hasBeginning": "2017-01-29",
+  "hasEnd": "",
+  "accrualPeriodicity": "maandelijks",
+  "publisher": "OIS",
+  "description": "Binnen de Gemeente Amsterdam wordt op verschillende niveaus inspecties uitgevoerd door verschillende directies en partijen. Bijvoorbeeld inspecties voor het dagelijkse onderhoud of technische inspecties voor vervangingsplannen.  Soms met een gelijke- methodiek en andere momenten een andere methodiek (CROW, NEN, etc.). Onder de dataset “inspecties” vind je diverse inspectie tabellen",
+  "status": "niet_beschikbaar",
+  "keywords": [
+    "inspecties",
+    "crow",
+    "beeldmeetlat",
+    "openbare ruimte",
+    "Stadswerken"
+  ],
+  "crs": "EPSG:28992",
+  "objective": "Bevat monitoring en inspectie data over de kwaliteit van de openbare ruimte van willekeurige meetlocaties binnen de gemeente Amsterdam (o.a. Weesp). ",
+  "temporalUnit": "seconden",
+  "contactPoint": {
+    "name": "",
+    "email": "datapunt@amsterdam.nl"
+  },
+  "tables": [
+    {
+      "id": "monitorbeeldkwaliteit",
+      "type": "table",
+      "version": "1.0.0", 
+      "title": "Monitorbeeldkwaliteit",
+      "description": "Deze tabel bevat de inspecties die zijn uitgevoerd voor het dagelijkse onderhoud Beheer Openbare Ruimte. Hiervoor wordt merendeels de CROW 380 methodiek gebruikt. Enkele inspecties worden met een ander doel uitgevoerd en hanteren daarom niet de CROW 380 methodiek.",
+      "provenance": "apt_inspecties",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "mainGeometry": "geometrie",
+        "identifier": "id",
+        "required": [
+          "id",
+          "schema"
+        ],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "string",
+            "description": "Unieke aanduiding van het record."
+          },
+          "environment": {
+            "type": "string",
+            "description": "Lijst bevat de volgende eigenschappen. environment Omgevingsparameter waarin de inspectie heeft plaatsgevonden (bijv. amsterdam, amsterdam2019, amsterdamcentrum, etc.)"
+          },
+          "inspectionRoundType": {
+            "type": "string",
+            "description": " "
+          },
+          "inspectionRoundId": {
+            "type": "string",
```

Comment:
De velden hebben geen title. Deze kunnen worden gebruikt als labels in dataportaal of andere frontends. Is mooier als ze er wel staan.

Reply:
Voor deze dataset zijn er geen plannen voor een mogelijke front-end applicatie waarbij de title relevant is. Wel voor de volgende dataset willen we title's aanvullen omdat er een front-end applicatie wordt gebouwd.